src/common/common_protocol.h

```diff
@@ -0,0 +1,41 @@
+#ifndef COMMON_PROTOCOL_H
+#define COMMON_PROTOCOL_H
+
+#include "format/common_generated.h"
+
+#include "common.h"
+
+/**
+ * Convert an object ID to a flatbuffer string.
+ *
+ * @param fbb Reference to the flatbuffer builder.
+ * @param object_id The object ID to be converted.
+ * @return The flatbuffer string contining the object ID.
+ */
+flatbuffers::Offset<flatbuffers::String> to_flat(
```

Comment:
I realize these methods were just moved around, but these names should probably be a bit more descriptive.